pkg/kubelet/api/v1alpha1/stats/types.go

```diff
@@ -117,10 +117,12 @@ type NetworkStats struct {
 type CPUStats struct {
 	// The time at which these stats were updated.
 	Time unversioned.Time `json:"time"`
-	// Total CPU usage (sum of all cores) averaged over the sample window.
-	// The "core" unit can be interpreted as CPU core-nanoseconds per second.
+	// Total CPU usage (sum of CPU time spent on each core) averaged over the
+	// sample window (default 10s).  The "nanocore" unit can be interpreted as
+	// CPU core-nanoseconds used per second.
```

Comment:
That was pseudo code for:
`UsageCoreNanoSeconds[current]` - The value of `UsageCoreNanoSeconds` in the most recent sample
`UsageCoreNanoSeconds[previous]` - The value of `UsageCoreNanoSeconds` in the previous sample

Feel free to change it to something more intuitive, I just thought it might be useful to explain how this field relates to `UsageCoreNanoSeconds`. This is moot if we decide to just remove this field (are you using it?)